msal4j-sdk/src/main/java/com/microsoft/aad/msal4j/AcquireTokenByAppProviderSupplier.java

```diff
@@ -87,6 +87,9 @@ public AuthenticationResult fetchTokenUsingAppTokenProvider(AppTokenProviderPara
                 .idToken(null)
                 .expiresOn(tokenProviderResult.getExpiresInSeconds())
                 .refreshOn(tokenProviderResult.getRefreshInSeconds())
+                .metadata(AuthenticationResultMetadata.builder()
```

Comment:
THis is ok, but how do you know it works? I.e. is there no testing that can be done here?